documentation/modules/creating-migration-plan-2-8.adoc

```diff
@@ -0,0 +1,70 @@
+. Select the source provider of the VMs you want to migrate.
++
+The *Select virtual machines* interface opens.
+. Select the VMs you want to migrate and click *Next*.
+endif::[]
+
+ifdef::vms[]
+= Creating a migration plan starting with specific VMs
+
+You can create a migration plan based on specific VMs, starting on the *Providers for virtualization* page. Note the specific options for migrations from VMware or {rhv-short} providers.
+
+.Procedure
+
+. In the {ocp} web console, click *Providers for virtualization*.
+. In the row of the appropriate source provider, click *VMs*.
++
+The *Virtual Machines* tab opens.
+. Select the VMs you want to migrate and click *Create migration plan*.
+endif::[]
++
+The *Create migration plan* pane opens. It displays the source provider's name and suggestions for a target provider and namespace, a network map, and a storage map.
+. Enter the *Plan name*.
+. Make any needed changes to the editable items.
+. Click *Add mapping* to edit a suggested network mapping or a storage mapping, or to add one or more additional mappings.
+. Click *Create migration plan*.
++
+{project-short} validates the migration plan and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. The details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} validates the plan again.
+
+. VMware source providers only (All optional):
+
+* *Preserving static IPs of VMs*: By default, virtual network interface controllers (vNICs) change during the migration process.  As a result, vNICs that are configured with a static IP linked to the interface name in the guest VM lose their IP. To avoid this, click the Edit icon next to *Preserve static IPs* and toggle the *Whether to preserve the static IPs* switch in the window that opens. Then click *Save*.
++
+{project-short} then issues a warning message about any VMs for which vNIC properties are missing. To retrieve any missing vNIC properties, run those VMs in vSphere in order for the vNIC properties to be reported to {project-short}.
+
+* *Entering a list of decryption passphrases for disks encrypted using Linux Unified Key Setup (LUKS)*: To enter a list of decryption passphrases for LUKS-encrypted devices, in the *Settings* section, click the Edit icon next to *Disk decryption passphrases*, enter the passphrases, and then click *Save*. You do not need to enter the passphrases in a specific order - for each LUKS-encrypted device, {project-short} tries each passphrase until one unlocks the device.
+
+* *Specifying a root device*: Applies to multi-boot VM migrations only. By default, {project-short} uses the first bootable device detected as the root device.
++
+To specify a different root device, in the *Settings* section, click the Edit icon next to *Root device* and choose a device from the list of commonly-used options, or enter a device in the text box.
```

Comment:
```suggestion
To specify a different root device, in the *Settings* section, click the *Edit* icon next to *Root device* and choose a device from the list of commonly-used options, or enter a device in the text box.
```

Reply:
Good catch. (Change made in https://github.com/kubev2v/forklift-documentation/pull/585).